TestResultSummaryService/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14
+FROM node:16
```

Comment:
Since we upgrade the node, we should upgrade to 18. 16 is out of active support and security support for 16 will end in Sep 2023.